Lib/fontTools/varLib/models.py

```diff
@@ -499,29 +500,34 @@ def getMasterScalars(self, targetLocation):
         return out
 
     @staticmethod
-    def interpolateFromDeltasAndScalars(deltas, scalars):
-        """Interpolate from deltas and scalars fetched from getScalars()."""
+    def interpolateFromValuesAndScalars(values, scalars):
+        """Interpolate from values and scalars coefficients."""
         v = None
-        assert len(deltas) == len(scalars)
-        for delta, scalar in zip(deltas, scalars):
+        assert len(values) == len(scalars)
+        for value, scalar in zip(values, scalars):
             if not scalar:
                 continue
-            contribution = delta * scalar
+            contribution = value * scalar
             if v is None:
                 v = contribution
             else:
                 v += contribution
         return v
 
+    @staticmethod
+    def interpolateFromDeltasAndScalars(deltas, scalars):
```

Comment:
yes, this one we can keep as an alias of `interpolateFromValuesAndScalars` for backward compat, but I don't think we also need `interpolateFromMastersAndMasterScalars`